lms/static/sass/elements/_course-card.scss

```diff
@@ -1,8 +1,45 @@
-.header-img{
-  @include float(left);
-  margin-right:20px;
-}
 .course-card{
-  height: 250px;
-  margin-top:20px;
+  width: 240px;
+  @include susy-media($bp-screen-sm) { width: 240px; }
+  @include susy-media($bp-screen-md) { width: 700px; }
+  @include susy-media($bp-screen-lg) { width: 920px; }
+  margin-left: $baseline*2 + px;
+  margin-bottom: $baseline + px;
+  .course-image-link{
+    .header-image{
+      width: 191px;
+    }
+  }
+  .course-title{
+    font-size: font-size(x-large);
+    font-weight: font-weight(normal);
+    margin-bottom: $baseline/4 + px;
+  }
+  .course-text{
+    color: $gray-d1;
+  }
+  .enrollment-info{
+    width: $baseline*10 + px;
+    text-align: center;
+    margin-bottom: $baseline/2 + px;
+  }
+  .enroll-button{
+    width: $baseline*10 + px;
+    text-align: center;
+    background-color: rgba(15, 124, 57, 1) !important;
+    border-color: rgba(15, 124, 57, 1) !important;
+    &:hover{
+      background-color: #0DB553 !important;
```

Comment:
Variables should be used for colours.
We should not need to use ```!important``` for new pages.